utensor_cgen/backend/utensor/code_generator/rearch/_code_generator.py

```diff
@@ -26,26 +26,19 @@ def __init__(self, config):
     self.src_fname = final_config['src_fname']
     self.header_fname = final_config['header_fname']
     self.params_dir = final_config['params_dir'].rstrip('/')
-    self.trans_methods = final_config['transform_methods']
     self.meta_data_pool_size = final_config['meta_data_pool_size']
     self.ram_data_pool_size = final_config['ram_data_pool_size']
     self.model_dir = final_config['model_dir'].rstrip('/')
-    self.save_graph = final_config['save_graph']
 
   def apply(self, ugraph):
```

Comment:
You mean the comments?

Reply:
I mean it might make things clearer with one subroutine for each chunklet For example:

`(ops, placeholders, tensor_var_map) = self._find_required_ops(...)`

Reply:
Right. I see.
Sure thing.
Let's me think about what are the proper names for these subroutines.

Reply:
Done.